src/components/QuizData.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import '../QuizData.css';
+
+export default class QuizData extends Component {
+  constructor() {
+    super();
+    this.state = {
+      quizzes: [],
+      questions: [],
+    };
+  }
+
+  componentDidMount() {
+    axios.get('/quizzes')
+      .then((response) => {
+        this.setState({
+          quizzes: response.data.quizzes,
+          questions: response.data.quizzes[0].questions
+        });
+      })
+      .catch((error) => {
+        console.log(error);
+    });
+  }
+
+  render() {
+    return (
+      <div>
+        {this.state.quizzes.map((quiz) => {
+          return (
+            <section key={quiz.id}>
+              <h1 className="pageTitle">{quiz.title}</h1>
+            </section>
+          );
+        })}
+        {this.state.questions.map((quiz) => {
+          return (
+            <section key={quiz.id} className="questionContainer">
```

Comment:
This is a lot of HTML we're returning when we map through our questions. It might make sense to make a `Question` component so that we can simply return something like:

```javascript
<Question title={question.title} key={question.id} answers={question.answers}/>
```